tasks/lib/phantomas.js

```diff
@@ -506,7 +506,7 @@ Phantomas.prototype.getImages = function() {
 
   return _.sortBy( files, _.bind( function( file ) {
     return +file.match(
-      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d-(\d*).png$/
+      /^screenshot-\d\d\d\d-\d\d-\d\dT\d\d-\d\d-\d\d-(\d*).png$/
```

Comment:
@macbre thanks for point this out.

This actually only for sorting the files properly. Every run has it's own directory as far as I remember, so there is no problem with it atm. :bowtie: 

Or do I miss the point?